compiler/plugins/target/AMD-AIE/iree-amd-aie/Target/AIETarget.cpp

```diff
@@ -172,7 +172,11 @@ LogicalResult AIETargetBackend::serializeExecutable(
   // generated by `aie2xclbin`
   SmallVector<std::string> entryPointNames;
   for (auto exportOp : variantOp.getExportOps()) {
-    entryPointNames.emplace_back(exportOp.getSymName());
+    // The xclbin kernel name, appended with instance name suffix (`:MLIRAIEV1`,
+    // 10 chars) is required by the xclbinutil to have a length smaller or equal
+    // to 64 right now. To have some additional wiggle room for suffix changes,
+    // we use the 48 first characters for the kernel name.
```

Comment:
```suggestion
    // we use the 48 first characters for the kernel name.
    // This is okay to do for now because we are only supporting single entry point.
```

Reply:
Done